### PR TITLE
blockdev_stream_speed:skip job running check for low speed set

### PR DESCRIPTION
**Pull request by aliang123**
Skip stream job running check for low speed set

Signed-off-by: Aihua Liang <aliang@redhat.com>
id:2078381

### CODE REVIEW

**Comment by aliang123** [comment]
depends on https://github.com/autotest/tp-qemu/pull/3311

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_stream_speed.q35: PASS (221.25 s)

**Comment by aliang123** [comment]
@zhencliu @MiriamDeng, please help to review, thanks.

File: qemu/tests/blockdev_stream_speed.py [hunk 1]
```diff
@@ -44,8 +44,22 @@ def _set_valid_speed(jobid, speed):
...
+                self.test.fail("Speed:%s is not set as expected:%s"
+                               % (job.get("speed"), speed))
+            ck_speed = self.params.get_numeric("check_speed")
+            uspeed = self.params.get_numeric("ulimit_speed")
+            if speed > ck_speed or speed == uspeed:
+                job_utils.check_block_jobs_running(
+                    self.main_vm, [self._job],
+                    self.params.get_numeric('job_running_timeout', 60)
+                    )
+
```
Comment (zhencliu) [inline qemu/tests/blockdev_stream_speed.py:59]: same comment as https://github.com/autotest/tp-qemu/pull/3311/files#r873481886

**Comment by zhencliu** [comment]
LGTM

**Comment by MiriamDeng** [comment]
LGTM
\ (1/1) Host_RHEL.m8.u7.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.7.0.ppc64le.io-github-autotest-qemu.blockdev_stream_speed: PASS (213.36 s)

**Pull request merged**